postgraphile/website/postgraphile/deploying-docker.md

```diff
@@ -59,7 +59,7 @@ There's a few critical things to keep in mind:
 To speed up builds, we recommend you pay attention to your `.dockerignore` file;
 here's an example to get you started:
 
-```
+```ini
```

Comment:
I don't think we should use `ini` for an ignore file. All the ignore files are somewhat equivalent: gitignore, eslintignore, prettierignore, etc - maybe there's a common syntax for "list of file globs"?

Reply:
I have found a more comprehensive list of languages: https://prismjs.com/#supported-languages